app/src/main/java/com/example/marvelapp/data/service/entity/CharacterEntity.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.marvelapp.data.service.entity
+
+class CharacterEntity(
+        val id: Int,
```

Comment:
These are all the character properties?

Reply:
No, in the future pr i will add the other properties, such as "comics" and "thumbnails". In this ocassion i just add the ones you saw, because i wanna test the api response.

Reply:
okay, but remember that you should not leave functionalities by the half